src/main/java/xyz/nucleoid/dungeons/dungeons/util/item/DgItemQuality.java

```diff
@@ -65,7 +65,7 @@ public String getTranslationKey() {
         return DgTranslationUtil.translationKeyOf("quality", id);
     }
 
-    public double getDamageMultiplier() {
+    public double getMultiplier() {
         return damageMultiplier;
```

Comment:
Does it make sense to also rename `damageMultiplier` here to `multiplier`?

Reply:
Yup, as the qualities are used for both armor & weapons, renaming it makes it more understandable, as using a `getDamageMultiplier()` in a `getArmorToughness` function can be concerning

Reply:
Cool, then ig we can rename the variable to just `multiplier` for clarity